src/patternfly/components/Drawer/drawer.scss

```diff
@@ -417,11 +417,6 @@ $pf-v6-c-drawer__panel--list--width: (25, 33, 50, 66, 75, 100);
   }
 }
 
-// Page level drawer - move to page component in breaking change
-.#{$drawer}__body > .#{$page}__main {
-  height: 100%;
-}
-
```

Comment:
Old v5 style, doesn't match the v6 structure.

Reply:
Is the drawer body container being removed completely?

Reply:
Not from the component, just from the page component structure when the page-main-container/page-main elements are a child of the drawer content. AFAIK the only way to get that structure in react is via `<Page notificationDrawer={...}>`. So we would just remove the wrapper element `<DrawerContentBody>` here - https://github.com/patternfly/patternfly-react/blob/5decd6a8bf9632288abdbc98d3cac1902303c04e/packages/react-core/src/components/Page/Page.tsx#L349

Reply:
Oh, it looks like it is just in relation to the page and does not impact the drawer component.